### PR TITLE
fix: override transparency mode from exists records on update (MAPCO-2796)

### PR DESCRIPTION
**Pull request by ronenkapelian**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

on update jobs-task, the transparency and tileOutputFormat params should be taken from original records and not from overseer post request

### CODE REVIEW

File: src/layers/models/layersManager.ts [hunk 1]
```diff
@@ -97,6 +97,10 @@ export class LayersManager {
       if (!existsInMapProxy) {
         throw new BadRequestError(`layer '${resourceId}-${productType}', is not exists on MapProxy`);
       }
+      //todo - override data from record - on future should not be provided from new route for update
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:100]: move https://github.com/MapColonies/discrete-overseer-creator/blob/81732564eb13c6e6e3aa7a7ad3590f08420ef0d7/src/layers/models/layersManager.ts#L74 line to "New" ingestion section

File: src/layers/models/layersManager.ts [hunk 2]
```diff
@@ -97,6 +97,10 @@ export class LayersManager {
...
+      data.metadata.transparency = record?.metadata.transparency;
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:101]: add warn log that the requested transparancey will be overrided by catalog

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: src/layers/models/layersManager.ts (hunk 1 above)
Comment (ronenkapelian) [reply src/layers/models/layersManager.ts:100]: done

**Review by asafMasa — APPROVED** [review]

**Pull request merged**